README.rst

```diff
@@ -75,197 +77,186 @@ Installation
+Here is a list of supported Cartridge CLI commands:
+
+* ``create``: create a new application from template.
+* ``build``: build an application for local development and testing.
+* ``start``: start one or more Tarantool instances.
+* ``stop``: stop one or more Tarantool instances.
+* ``status``: get the status of one or more current instances.
+* ``log``: get logs for one or more instances.
+* ``clean``: clean files for one or more instances.
+* ``pack``: pack the application into a distributable bundle.
+* ``repair``: patch cluster configuration files.
+* `admin <doc/admin.rst>`__: call an admin function provided by the application.
+* `replicasets <doc/replicasets.rst>`__: manage cluster replica sets running locally.
+* `enter <doc/connect.rst>`__ and `connect <doc/connect.rst>`__: connect to a running instance.
+* `failover <doc/failover.rst>`_: manage cluster failover.
+
+You can control output verbosity with these global flags:
+
+* ``verbose``: displays additional log messages as well as
+  commands/docker output, such as the output of ``tarantoolctl rocks make`` or ``docker build``.
+* ``debug``: works the same as verbose, but temporary files and
+  directories aren't removed during command execution.
+* ``quiet``: hides all logs, only displays error messages.
+
+Application lifecycle
+~~~~~~~~~~~~~~~~~~~~~
 
 In a nutshell:
 
-1. `Create <Creating an application from template_>`_
-   an application (e.g. ``myapp``) from template:
+1. :ref:`Create an application <cartridge-cli-creating_an_application_from_template>`
+   (for example, ``myapp``) from a template:
 
    .. code-block:: bash
 
        cartridge create --name myapp
        cd ./myapp
 
-2. `Build <Building an application_>`_ the application
+2. :ref:`Build the application <cartridge-cli-building-the-application>`
    for local development and testing:
 
    .. code-block:: bash
 
        cartridge build
 
-3. `Run <Starting/stopping an application locally_>`_
-   instances locally:
+3. :ref:`Run instances locally <cartridge-cli-starting-the-application-locally>`:
 
    .. code-block:: bash
 
        cartridge start
        cartridge stop
 
-4. `Pack <Packing an application_>`_ the application into
-   a distributable (e.g. into an RPM package):
+4. :ref:`Pack the application <cartridge-cli-packaging-the-application>`
+   into a distributable (like an RPM package):
 
    .. code-block:: bash
 
        cartridge pack rpm
 
 .. _cartridge-cli-creating_an_application_from_template:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Creating an application from template
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Creating an application from a template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create an application from the Cartridge template, say this in any directory:
+To create an application from a Cartridge template, run ``cartridge create`` in any directory:
```

Comment:
From arbitrary Cartirdge template? I guess the sentence is about certain template, generated by cartridge cli.

Reply:
Here I'm in doubt. We didn't mention that there is a way to choose a custom template, but it is in several lines below. So up to you.